use/adopter/whatsapp-flow-for-uci.md

```diff
@@ -0,0 +1,218 @@
+# Whatsapp Flow (Netcore)
+
+## Types of Bots:
+
+1. **Broadcast Bot:** This type of bot is used to send a single message to a large group of people. It doesn't expect a reply to the message.
+
+2. **Conversation Bot:** This type of bot is designed for a proper conversation flow with a user. It facilitates a two-way conversation that is regulated by the ODK form.
+
+## Broadcast Flow
+
+This is a 4 step process.
+
+1. Create conversation logic: 
+
+```bash
+curl --location 'http://localhost:3002/admin/conversationLogic' \
+--header 'admin-token: dR67yAkMAqW5P9xk6DDJnfn6KbD4EJFVpmPEjuZMq44jJGcj65' \
+--header 'Content-Type: application/json' \
+--header 'Cookie: fusionauth.locale=en_US; fusionauth.sso=AgOat0GjncGOHhPpH_HuL9QQqnfMitd15O-ofS-uTcdA' \
+--data '{
+    "data": {
+        "id": null,
+        "name": "Netcore Demo",
+        "description": "Netcore Demo",
+        "transformers": [
+            {
+                "id": "774cd134-6657-4688-85f6-6338e2323dde",
+                "meta": {
+                    "body": "Hello ${name}-${phoneNo}, Netcore Test Notification Body demo dry run.",
+                    "type": "broadcast",
+                    "title": "Netcore Demo",
+                    "params": [
+                        "name",
+                        "phoneNo"
+                    ],
+                    "templateType": "JS_TEMPLATE_LITERALS"
+                }
+            }
+        ],
+        "adapter": "44a9df72-3d7a-4ece-94c5-98cf26307323"
+    }
+}'
+```
+
+This will create a conversation logic for the broadcast flow. We don't need an ODK form here as this is not a multi-message flow. The conversation logic for this type of bot only has a single broadcast message which is defined in the meta of the request.
```

Comment:
Please change the words `multi-message` as it is ambiguous. We can reframe the statement as:

`We don't need an ODK form here as this is not a two-way conversation flow that is driven by ODK logic.`